openfga_sdk/configuration.py

```diff
@@ -56,13 +56,22 @@ def max_retry(self):
         """
         Return the maximum number of retry
         """
+        if self._max_retry > 15:
+            return 15
+
         return self._max_retry
 
     @max_retry.setter
     def max_retry(self, value):
         """
         Update the maximum number of retry
         """
+        if not isinstance(value, int) or value < 0:
+            raise ValueError("max_retry must be an integer greater than or equal to 0")
+
+        if value > 15:
+            value = 15
```

Comment:
This seems a bit misleading, can we do like the other SDKs and through an error here?

Otherwise might think that by increasing it, they might retry more often

Reply:
Also would be great if we can use the same error message 🙏

Reply:
![tv-show-of-course-gif-by-laff](https://github.com/openfga/python-sdk/assets/3093/f69918d5-2d53-4fec-97b2-d2bdb205cc2d)